todoman/ui.py

```diff
@@ -91,7 +91,8 @@ def __init__(self, todo, databases, formatter):
         self._help_text = urwid.Text(
             '\n\nGlobal:\n'
             ' F1: Toggle help\n'
-            ' Ctrl-C: Cancel\n\n'
+            ' Ctrl-C: Cancel\n'
+            ' Ctrl-S: Save (only works if not a shell shortcut already)\n\n'
```

Comment:
All the good ones seem taken I don't think we'll find anything that makes sense. Some options: 

* <kbd>ctrl</kbd> + <kbd>q</kbd>: "save and Quit".
* <kbd>ctrl</kbd> + <kbd>t</kbd>: save (not real relation between action and key).
* <kbd>ctrl</kbd> + <kbd>V</kbd>: "saVe"